### ISSUE TITLE
Solo track is not unique after copy/paste

### ISSUE THREAD

**Issue by Paul-Licameli** [audacity/audacity#2791]
**Describe the bug**
At most one track at a time should ever have the Solo button down.  Copy/paste doesn't respect that.

**To Reproduce**
1. Be sure Tracks Behaviors preferences chooses Simple for the Solo button (meaning, Solo is exclusive)
2. Generate an audio track, make it Solo
3. Select all, Copy, de-select all, Paste3. 
4. Do the same with a MIDI track, after #2790 is fixed, to be sure there is no similar bug

**Actual behavior**
Both tracks are solo

**Expected behavior**
Only one track in the entire project should ever be solo when the preference is default.  (But if it is not, Solo can be non-exclusive; then Solo status should be copied.)

Observe that just depressing a Solo button, with the default preference chosen, causes others to pop up.  (But there is also the little known Shift-Click which does the opposite of the preference!)

**Screenshots**
If applicable, add screenshots/video to help explain your problem.

**Additional information (please complete the following information):**
 - OS: [e.g. Mac High Sierra, Ubuntu 18.04, Windows 10]
 - Version [e.g. Audacity 2.5.1]
 - (if relevant) Sound Device [e.g. Windows WASAPI, USB Sound card]

**Additional context**

**Comment by petersampsonaudacity** [comment]
Confirmed on W10
![image](https://user-images.githubusercontent.com/50205984/162576152-4ea4b112-0ad0-4b61-b199-57c00090b51e.png)

**Comment by micpap25** [comment]
I'd like to work on resolving this issue.
I'm also creating a new issue based on the issues with the Shift-Click functionality I've noticed while experimenting. However, the issue of copy-pasting not carrying over is separate so I'll attempt to handle that here.

**Comment by Paul-Licameli** [comment]
Please don't.

There is another fix in progress for this.

**Comment by micpap25** [comment]
OK.
